### PR TITLE
Docker dev setup

### PR DESCRIPTION
**Pull request by t0mk**
My basic Docker dev setup. Code is host-mounted. Cron runs in the same container as Django. Simple scirpt in web container startup waits for postgres, and then attempts to do what you suggest to initialize in the readme.

Merge if you like. Let me know if you think there's sth to fix or improve.

### CODE REVIEW

**Comment by owocki** [comment]
Thanks for the PR Tom.  I'm going to test this out tonight on my home machine.

**Comment by owocki** [comment]
having some trouble getting this up & running locally via docker (probably due to my docker-noob-ness).  Have you requested an invite to our slack?  I'd like to hear how you've been using the repo, and also talk through an issue or two I'm having.

https://github.com/owocki/pytrader/issues/23

**Comment by Snipa22** [comment]
This works fine on my systems for development.

The suggestions I'd make are as follows:
Add examples for the .env and the docker-compose.yml files, then put them into the .gitignore, so we can do local customization to these easier.

**Comment by owocki** [comment]
@t0mk what do you think? if you want to make those changes, and then fix the merge conflicts, Ill happily merge this into the repo.

**Comment by t0mk** [comment]
I'll do that later today!

**Comment by t0mk** [comment]
Done.

**Comment by t0mk** [comment]
Fixed some issues, now it's alright.

File: requirements.txt [hunk 1]
```diff
@@ -1,9 +1,11 @@
 django==1.8.2
-scipy==0.13.3
-matplotlib==1.3.1
-PyBrain==0.3
-psycopg2==2.6.1
-django-chartit==0.1
-simplejson==3.8.2
-Cython==0.23.5
-sklearn==0.0
+scipy
```
Comment (owocki) [inline requirements.txt]: i've always specified pip versions for my teammates. helps eliminate differences in our testing / production environments.  was this change intentional?  if so, why is it good not to specify versions?

**Comment by owocki** [comment]
erg, some recent PR merges have created merge conflicts.

@t0mk @Snipa22 i've given you push access to the repository, so you (should) now have the ability to merge changes to master.  for this PR, just need to correct the merge conflicts.  for future PRs, my only ask to start is that we all get aligned on changes to master via a pull request.

File: requirements.txt (hunk 1 above)
Comment (t0mk) [inline requirements.txt]: Yes, it was intentional - the pip call doesn't succeed with the versions specified. The numpy setup.py failed.

Comment (owocki) [inline requirements.txt]: i believe there are some `apt-get` dependancies here too.  need to find/ document those.

Comment (t0mk) [inline requirements.txt]: apt-get deps are not necessary so please don't bother. It's built based on official Python 2.7 image, which is some sort of debian.

Comment (t0mk) [inline requirements.txt]: I think the problem while building was that that version of scipy was not possible to be built with the newest numpy (and I didn't know which version of numpy you had, because it's not in the requirements). So I just dropped all the versions (except for Django because that broke down the startup) and build it. Now, I think I found the proper version of numpy so let's see.

It's not instant to see if the things work out, because install of numpy and then scipy takes some time (~20 miuntes on my crappy laptop).

**Comment by NDuma** [comment]
What do I need to do to load this into cloud.docker.com?

**Comment by owocki** [comment]
good question  .. might be good to open up an issue and discuss there.

**Comment by owocki** [comment]
i'd bet with a little googling you could come up with an approach that is 80% of the way there.

**Comment by NDuma** [comment]
I'm on it.
